README.md

```diff
@@ -1 +1,47 @@
-# Math-Magicians
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Math Magicians
+
+> Set up a React app using React documentation.
+
+
+## Built With
+
+- HTML, CSS & JavaScript
+- React components.
+- React props.
+- React state
+
+## Getting Started
+
+- npx create-react-app my-app
+- cd my-app
+- npm start
```

Comment:
- **[OPTIONAL]** The previous reviewer meant that you should include steps on how to set up this very app, not create a new react app. See example
  ```
  - git clone https://github.com/gasangw/Math-Magicians.git
  - cd Math-Magicians
  - npm install
  - npm start
  ```

Reply:
Thank you so much @DammyShittu